samples/grid-pro/demo/todo-app/demo.js

```diff
@@ -0,0 +1,266 @@
+            },
+            validationRules: ['notEmpty']
+        }
+    }
+}, {
+    id: 'Task',
+    cells: {
+        editMode: {
+            validationRules: ['notEmpty']
+        }
+    }
+}, {
+    id: 'Notes'
+}, {
+    id: 'Due date',
+    dataType: 'datetime',
+    cells: {
+        formatter: function () {
+            return this.value ? new Date(this.value).toLocaleDateString(
+                navigator.language
+            ) : '';
+        },
+        editMode: {
+            renderer: {
+                type: 'dateInput'
+            }
+        }
+    }
+}, {
+    id: 'Priority',
+    className: 'centered',
+    width: 120,
+    cells: {
+        formatter: function () {
+            const map = {
+                1: '🟢',
+                2: '🟡',
+                3: '🔴'
+            };
+            return map[this.value] || '';
+        },
+        editMode: {
+            renderer: {
+                type: 'select',
+                options: priority
+            }
+        }
+    },
+    header: {
+        className: 'centered-header'
+    }
+}];
+
+// Grid setup
+Grid.grid('container', {
+    time: {
+        locale: ''
+    },
+    dataTable: {
+        columns: {
+            Done: [false, false, false, false],
+            Category: ['CL', 'SH', 'CL', 'GR'],
+            Task: [
+                'Do laundry', 'Buy groceries', 'Clean kitchen', 'Water plants '
+            ],
+            Notes: [
+                'Use gentle cycle', 'Get milk and eggs', '',
+                // eslint-disable-next-line max-len
+                'In the kitchen use 2 cups of water, in the living room use 1 cup, do not water the cactus in the garden'
+            ],
+            'Due date': [
+                Date.UTC(2025, 6, 1),
+                Date.UTC(2025, 6, 2),
+                Date.UTC(2025, 6, 3),
+                Date.UTC(2025, 6, 4)
+            ],
+            Priority: [1, 3, 2, 1]
+        }
+    },
+    columnDefaults: {
+        cells: {
+            editMode: {
+                enabled: true
+            }
+        }
+    },
+    columns: columns,
+    credits: {
+        enabled: false
+    }
+});
+Grid.grid('container-done', {
+    dataTable: {
+        columns: {
+            Done: [true, true, true],
+            Category: ['CK', 'GR', 'MT'],
+            Task: ['Cook dinner', 'Trim hedges', 'Fix leaking tap'],
+            Notes: [
+                'Try new pasta recipe', 'Backyard only', 'Check under sink'
+            ],
+            'Due date': [
+                Date.UTC(2025, 6, 8),
+                Date.UTC(2025, 6, 9),
+                Date.UTC(2025, 6, 10)
+            ],
+            Priority: [2, 1, 3]
+        }
+    },
+    columnDefaults: {
+        cells: {
+            editMode: {
+                enabled: true
+            }
+        }
+    },
+    columns: columns
+});
+
+// Custom events
+function addCustomEvents(isTodoGrid) {
+    const sourceGrid = Grid.grids[isTodoGrid ? 0 : 1];
```

Comment:
`!!isTodoGrid` would not be enough here?

Reply:
no, we need number to index the element in the `grids` array